README.md

```diff
@@ -18,7 +18,7 @@ For a video overview of PowerToys, including install steps and a walkthrough of
 
 ### Color Picker
 
-[<img align="left" src="https://aka.ms/powerToysColorPickerImageSmall" />](https://aka.ms/PowerToysOverview_ColorPicker) [Color Picker](https://aka.ms/PowerToysOverview_ColorPicker) is a simple and quick system-wide color picker with <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>C</kbd>. Color Picker allows to pick colors from any currently running application and automatically copies the HEX or RGB values to your clipboard. This code is based on [Martin Chrzan's Color Picker](https://github.com/martinchrzan/ColorPicker).
+[<img align="left" src="https://aka.ms/powerToysColorPickerImageSmall" />](https://aka.ms/PowerToysOverview_ColorPicker) [Color Picker](https://aka.ms/PowerToysOverview_ColorPicker) is a simple and quick system-wide color picker with <kbd>Win</kbd>+<kbd>Shift</kbd>+<kbd>C</kbd>. Color Picker allows to pick colors from any currently running application and automatically copies it in a configurable format to your clipboard. Color picker also contains editor which let you see a history of picked color, fine-tune selected color and copy different string representations of your color. This code is based on [Martin Chrzan's Color Picker](https://github.com/martinchrzan/ColorPicker) and [niels9001's UX concept](https://github.com/niels9001/ColorPickerUX).
```

Comment:
"Color picker also contains editor which let you see a history of picked color, fine-tune selected color and copy different string representations of your color."

=>

"Color Picker also contains an editor that shows a history of previously picked colors, allows you to fine-tune the selected color and to copy different string representations."


"and [niels9001's UX concept](https://github.com/niels9001/ColorPickerUX)" 

=>

"and [Niels Laute's UX concept](https://medium.com/@Niels9001/a-fluent-color-meter-for-powertoys-20407ededf0c)"

Reply:
Ugh, it's formatted.. I changed the link, instead of the repo to the blog post: https:// medium.com/@Niels9001/a-fluent-color-meter-for-powertoys-20407ededf0c

Reply:
@niels9001
We should add this link to your blog in settings too.